Domain/sdLitica.Entities/Analytics/AnalyticsOperationRequest.cs

```diff
@@ -14,30 +14,24 @@ public enum OperationStatus
     }
 
 
-    //  At this moment, also used as its model in requests, messages. 
-
     /// <summary>
     /// Entity for metadata of analytics operation.
     /// </summary>
-    public class AnalyticsOperation: Entity
+    public class AnalyticsOperationRequest : Entity
```

Comment:
will try to change to more reasonable name. is suffix "Entity" okay? e.g. AnalyticsOperationRequestEntity/AnalyticsRequestEntity

Reply:
I dont think so, because the remainder entitties wont have this suffix. And, it is not nice for you to change the remainder classes name to use Entity as suffix.

Maybe a simple thing to do is to use another synonym to represent those tables/entities ```AnalyticsOperationRequest``` and ```AnalyticsRequest```. I mean, just change how we see theses operations regarding the domain of the application. For example, just simple as ```Analytics``` and ```AnalyticsOperation``` presuming that these are request from user, or ```UserAnalytics``` and ```UserAnalyticsOperation```.

Reply:
renamed to UserAnalyticsOperation